tools/ci_build/github/linux/run_dockerbuild.sh

```diff
@@ -69,6 +70,27 @@ else
             docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg OS_VERSION=16.04 --build-arg PYTHON_VERSION=${PYTHON_VER} -f Dockerfile.ubuntu .
         fi
     fi
+else
+    if [ $BUILD_DEVICE = "gpu" ]; then
+        IMAGE="ubuntu18.04-$CUDA_VER"
+        DOCKER_FILE=Dockerfile.ubuntu_gpu
+        if [ $CUDA_VER = "cuda9.1-cudnn7.1" ]; then
+        DOCKER_FILE=Dockerfile.ubuntu_gpu_cuda9
+        fi
+        docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
+    elif [ $BUILD_DEVICE = "tensorrt" ]; then
+        # TensorRT container release 19.09
+        IMAGE="ubuntu18.04-cuda10.1-cudnn7.6-tensorrt6.0"
+        DOCKER_FILE=Dockerfile.ubuntu_tensorrt
+        docker build -t "onnxruntime-$IMAGE" --build-arg BUILD_USER=onnxruntimedev --build-arg BUILD_UID=$(id -u) --build-arg OS_VERSION=18.04 --build-arg PYTHON_VERSION=${PYTHON_VER} -f $DOCKER_FILE .
+    elif [ $BUILD_DEVICE = "openvino" ]; then
+        IMAGE="ubuntu18.04-openvino"
```

Comment:
where is there ubuntu18.04 for openvino? is there such an image?

Reply:
I see openvino can run on ubuntu18.04 here http://docs.openvinotoolkit.org/latest/_docs_install_guides_installing_openvino_linux.html.

Reply:
yes, but we are using ubuntu 16.04 https://github.com/microsoft/onnxruntime/blob/master/tools/ci_build/github/linux/docker/Dockerfile.ubuntu_openvino#L1
you can leave this. let's just try to get rid of any hardcoding of versions in this file.
ideally use $BUILD_OS everywhere possible.